### ISSUE TITLE
OrientDB v1.7.10 - Wrong parse xml handler for Hazelcast - Bootstrap

### ISSUE THREAD

**Issue by Ostico** [orientechnologies/orientdb#3324]
Running ```server.sh```, the server starts in distributed mode ( and does not accept connections from clients ) with the default ```orientdb-server-config.xml```, even when the config row is changed from
```xml
<parameter value="${distributed}" name="enabled"/>
```
to 
```xml
<parameter value="false" name="enabled"/>
```
```
WARN Distributed server is not yet ONLINE (status=OFFLINE), reject incoming connection from /127.0.0.1:43198 [OServerNetworkListener]
```

To start the server in standalone mode all the hazelcast handler in ```orientdb-server-config.xml``` must be removed/commented.

Should be documented/written or fixed.

**Comment by lvca** [comment]
I can't reproduce it, OrientDB standalone server starts with no problems.

**Comment by Ostico** [comment]
In my case it starts, but does not accept remote connections.

I tried with this [https://github.com/orientechnologies/orientdb/releases/tag/1.7.10](https://github.com/orientechnologies/orientdb/releases/tag/1.7.10) in my local server and on travis-ci.org

This is my log:
```

2015-01-02 21:14:50:153 INFO Loading configuration from: /var/www/PhpOrient/ci/environment/orientdb-community-1.7.10/config/orientdb-server-config.xml... [OServerConfigurationLoaderXml]
2015-01-02 21:14:50:155 FINE Trying to locate {0} [bind]
2015-01-02 21:14:50:156 FINE   not found [bind]
2015-01-02 21:14:50:156 FINE Checking system property {0} [bind]
2015-01-02 21:14:50:156 FINE   not found [bind]
2015-01-02 21:14:50:156 FINE Checking system property {0} [bind]
2015-01-02 21:14:50:156 FINE   not found [bind]
2015-01-02 21:14:50:156 FINE Unable to find from OSGi: {0} [bind]
2015-01-02 21:14:50:157 FINE Checking META-INF/services [bind]
2015-01-02 21:14:50:158 FINE Unable to find: {0} [bind]
2015-01-02 21:14:50:158 FINE Trying to create the platform default provider [bind]
2015-01-02 21:14:50:158 FINE Trying to load {0} [bind]
2015-01-02 21:14:50:159 FINE loaded {0} from {1} [bind]
2015-01-02 21:14:50:160 FINE Property com.sun.xml.internal.bind.XmlAccessorFactoryis not active.  Using JAXB's implementation [ContextFactory]
2015-01-02 21:14:50:291 FINE Using optimized Accessor for public java.util.List com.orientechnologies.orient.server.config.OServerConfiguration.handlers [OptimizedAccessorFactory]
2015-01-02 21:14:50:293 FINE Using optimized Accessor for public com.orientechnologies.orient.server.config.OServerParameterConfiguration[] com.orientechnologies.orient.server.config.OServerHandlerConfiguration.parameters [OptimizedAccessorFactory]
2015-01-02 21:14:50:296 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerParameterConfiguration.value [OptimizedAccessorFactory]
2015-01-02 21:14:50:296 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerParameterConfiguration.name [OptimizedAccessorFactory]
2015-01-02 21:14:50:298 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerHandlerConfiguration.clazz [OptimizedAccessorFactory]
2015-01-02 21:14:50:299 FINE Using optimized Accessor for public java.util.List com.orientechnologies.orient.server.config.OServerConfiguration.hooks [OptimizedAccessorFactory]
2015-01-02 21:14:50:299 FINE Using optimized Accessor for public com.orientechnologies.orient.server.config.OServerParameterConfiguration[] com.orientechnologies.orient.server.config.OServerHookConfiguration.parameters [OptimizedAccessorFactory]
2015-01-02 21:14:50:300 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerHookConfiguration.clazz [OptimizedAccessorFactory]
2015-01-02 21:14:50:301 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerHookConfiguration.position [OptimizedAccessorFactory]
2015-01-02 21:14:50:302 FINE Using optimized Accessor for public com.orientechnologies.orient.server.config.OServerNetworkConfiguration com.orientechnologies.orient.server.config.OServerConfiguration.network [OptimizedAccessorFactory]
2015-01-02 21:14:50:303 FINE Using optimized Accessor for public java.util.List com.orientechnologies.orient.server.config.OServerNetworkConfiguration.sockets [OptimizedAccessorFactory]
2015-01-02 21:14:50:304 FINE Using optimized Accessor for public com.orientechnologies.orient.server.config.OServerParameterConfiguration[] com.orientechnologies.orient.server.config.OServerSocketFactoryConfiguration.parameters [OptimizedAccessorFactory]
2015-01-02 21:14:50:305 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerSocketFactoryConfiguration.implementation [OptimizedAccessorFactory]
2015-01-02 21:14:50:305 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerSocketFactoryConfiguration.name [OptimizedAccessorFactory]
2015-01-02 21:14:50:306 FINE Using optimized Accessor for public java.util.List com.orientechnologies.orient.server.config.OServerNetworkConfiguration.protocols [OptimizedAccessorFactory]
2015-01-02 21:14:50:309 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerNetworkProtocolConfiguration.implementation [OptimizedAccessorFactory]
2015-01-02 21:14:50:310 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerNetworkProtocolConfiguration.name [OptimizedAccessorFactory]
2015-01-02 21:14:50:311 FINE Using optimized Accessor for public java.util.List com.orientechnologies.orient.server.config.OServerNetworkConfiguration.listeners [OptimizedAccessorFactory]
2015-01-02 21:14:50:312 FINE Using optimized Accessor for public com.orientechnologies.orient.server.config.OServerCommandConfiguration[] com.orientechnologies.orient.server.config.OServerNetworkListenerConfiguration.commands [OptimizedAccessorFactory]
2015-01-02 21:14:50:313 FINE Using optimized Accessor for public com.orientechnologies.orient.server.config.OServerEntryConfiguration[] com.orientechnologies.orient.server.config.OServerCommandConfiguration.parameters [OptimizedAccessorFactory]
2015-01-02 21:14:50:314 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerEntryConfiguration.value [OptimizedAccessorFactory]
2015-01-02 21:14:50:315 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerEntryConfiguration.name [OptimizedAccessorFactory]
2015-01-02 21:14:50:316 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerCommandConfiguration.implementation [OptimizedAccessorFactory]
2015-01-02 21:14:50:316 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerCommandConfiguration.pattern [OptimizedAccessorFactory]
2015-01-02 21:14:50:318 FINE Using optimized TransducedAccessor for com.orientechnologies.orient.server.config.OServerCommandConfiguration#stateful [OptimizedTransducedAccessorFactory]
2015-01-02 21:14:50:319 FINE Using optimized Accessor for public com.orientechnologies.orient.server.config.OServerParameterConfiguration[] com.orientechnologies.orient.server.config.OServerNetworkListenerConfiguration.parameters [OptimizedAccessorFactory]
2015-01-02 21:14:50:320 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerNetworkListenerConfiguration.protocol [OptimizedAccessorFactory]
2015-01-02 21:14:50:321 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerNetworkListenerConfiguration.socket [OptimizedAccessorFactory]
2015-01-02 21:14:50:322 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerNetworkListenerConfiguration.portRange [OptimizedAccessorFactory]
2015-01-02 21:14:50:323 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerNetworkListenerConfiguration.ipAddress [OptimizedAccessorFactory]
2015-01-02 21:14:50:324 FINE Using optimized Accessor for public com.orientechnologies.orient.server.config.OServerStorageConfiguration[] com.orientechnologies.orient.server.config.OServerConfiguration.storages [OptimizedAccessorFactory]
2015-01-02 21:14:50:326 FINE Using optimized TransducedAccessor for com.orientechnologies.orient.server.config.OServerStorageConfiguration#loadOnStartup [OptimizedTransducedAccessorFactory]
2015-01-02 21:14:50:327 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerStorageConfiguration.userPassword [OptimizedAccessorFactory]
2015-01-02 21:14:50:328 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerStorageConfiguration.userName [OptimizedAccessorFactory]
2015-01-02 21:14:50:329 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerStorageConfiguration.path [OptimizedAccessorFactory]
2015-01-02 21:14:50:330 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerStorageConfiguration.name [OptimizedAccessorFactory]
2015-01-02 21:14:50:331 FINE Using optimized Accessor for public com.orientechnologies.orient.server.config.OServerUserConfiguration[] com.orientechnologies.orient.server.config.OServerConfiguration.users [OptimizedAccessorFactory]
2015-01-02 21:14:50:332 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerUserConfiguration.resources [OptimizedAccessorFactory]
2015-01-02 21:14:50:333 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerUserConfiguration.password [OptimizedAccessorFactory]
2015-01-02 21:14:50:334 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerUserConfiguration.name [OptimizedAccessorFactory]
2015-01-02 21:14:50:335 FINE Using optimized Accessor for public com.orientechnologies.orient.server.config.OServerSecurityConfiguration com.orientechnologies.orient.server.config.OServerConfiguration.security [OptimizedAccessorFactory]
2015-01-02 21:14:50:337 FINE Using optimized Accessor for public java.util.List com.orientechnologies.orient.server.config.OServerSecurityConfiguration.users [OptimizedAccessorFactory]
2015-01-02 21:14:50:339 FINE Using optimized Accessor for public java.util.List com.orientechnologies.orient.server.config.OServerSecurityConfiguration.resources [OptimizedAccessorFactory]
2015-01-02 21:14:50:340 FINE Using optimized Accessor for public com.orientechnologies.orient.server.config.OServerEntryConfiguration[] com.orientechnologies.orient.server.config.OServerConfiguration.properties [OptimizedAccessorFactory]
2015-01-02 21:14:50:394 INFO OrientDB Server v1.7.10 (build 0) is starting up... [OServer]
2015-01-02 21:14:50:401 INFO Databases directory: /var/www/PhpOrient/ci/environment/orientdb-community-1.7.10/databases [OServer]
2015-01-02 21:14:50:440 INFO Listening binary connections on 0.0.0.0:2424 (protocol v.21, socket=default) [OServerNetworkListener]
2015-01-02 21:14:50:441 INFO Listening http connections on 0.0.0.0:2480 (protocol v.10, socket=default) [OServerNetworkListener]
2015-01-02 21:14:50:454 INFO Installing dynamic plugin 'studio-1.7.8.zip'... [OServerPluginManager]
2015-01-02 21:14:50:533 INFO Installing GREMLIN language v.2.5.0 - graph.pool.max=50 [OGraphServerHandler]
2015-01-02 21:14:50:550 WARN [OVariableParser.resolveVariables] Error on resolving property: distributed [orientechnologies]
2015-01-02 21:14:50:551 WARN Generating new node name for current node: node1420229690551 [OHazelcastPlugin]
2015-01-02 21:14:50:551 FINE Trying to locate {0} [bind]
2015-01-02 21:14:50:551 FINE   not found [bind]
2015-01-02 21:14:50:551 FINE Checking system property {0} [bind]
2015-01-02 21:14:50:552 FINE   not found [bind]
2015-01-02 21:14:50:552 FINE Checking system property {0} [bind]
2015-01-02 21:14:50:552 FINE   not found [bind]
2015-01-02 21:14:50:552 FINE Unable to find from OSGi: {0} [bind]
2015-01-02 21:14:50:552 FINE Checking META-INF/services [bind]
2015-01-02 21:14:50:553 FINE Unable to find: {0} [bind]
2015-01-02 21:14:50:553 FINE Trying to create the platform default provider [bind]
2015-01-02 21:14:50:553 FINE Trying to load {0} [bind]
2015-01-02 21:14:50:554 FINE loaded {0} from {1} [bind]
2015-01-02 21:14:50:554 FINE Property com.sun.xml.internal.bind.XmlAccessorFactoryis not active.  Using JAXB's implementation [ContextFactory]
2015-01-02 21:14:50:578 FINE Using optimized Accessor for public java.util.List com.orientechnologies.orient.server.config.OServerConfiguration.handlers [OptimizedAccessorFactory]
2015-01-02 21:14:50:578 FINE Using optimized Accessor for public com.orientechnologies.orient.server.config.OServerParameterConfiguration[] com.orientechnologies.orient.server.config.OServerHandlerConfiguration.parameters [OptimizedAccessorFactory]
2015-01-02 21:14:50:579 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerParameterConfiguration.value [OptimizedAccessorFactory]
2015-01-02 21:14:50:580 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerParameterConfiguration.name [OptimizedAccessorFactory]
2015-01-02 21:14:50:580 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerHandlerConfiguration.clazz [OptimizedAccessorFactory]
2015-01-02 21:14:50:581 FINE Using optimized Accessor for public java.util.List com.orientechnologies.orient.server.config.OServerConfiguration.hooks [OptimizedAccessorFactory]
2015-01-02 21:14:50:582 FINE Using optimized Accessor for public com.orientechnologies.orient.server.config.OServerParameterConfiguration[] com.orientechnologies.orient.server.config.OServerHookConfiguration.parameters [OptimizedAccessorFactory]
2015-01-02 21:14:50:582 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerHookConfiguration.clazz [OptimizedAccessorFactory]
2015-01-02 21:14:50:583 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerHookConfiguration.position [OptimizedAccessorFactory]
2015-01-02 21:14:50:584 FINE Using optimized Accessor for public com.orientechnologies.orient.server.config.OServerNetworkConfiguration com.orientechnologies.orient.server.config.OServerConfiguration.network [OptimizedAccessorFactory]
2015-01-02 21:14:50:584 FINE Using optimized Accessor for public java.util.List com.orientechnologies.orient.server.config.OServerNetworkConfiguration.sockets [OptimizedAccessorFactory]
2015-01-02 21:14:50:585 FINE Using optimized Accessor for public com.orientechnologies.orient.server.config.OServerParameterConfiguration[] com.orientechnologies.orient.server.config.OServerSocketFactoryConfiguration.parameters [OptimizedAccessorFactory]
2015-01-02 21:14:50:586 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerSocketFactoryConfiguration.implementation [OptimizedAccessorFactory]
2015-01-02 21:14:50:586 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerSocketFactoryConfiguration.name [OptimizedAccessorFactory]
2015-01-02 21:14:50:587 FINE Using optimized Accessor for public java.util.List com.orientechnologies.orient.server.config.OServerNetworkConfiguration.protocols [OptimizedAccessorFactory]
2015-01-02 21:14:50:587 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerNetworkProtocolConfiguration.implementation [OptimizedAccessorFactory]
2015-01-02 21:14:50:588 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerNetworkProtocolConfiguration.name [OptimizedAccessorFactory]
2015-01-02 21:14:50:589 FINE Using optimized Accessor for public java.util.List com.orientechnologies.orient.server.config.OServerNetworkConfiguration.listeners [OptimizedAccessorFactory]
2015-01-02 21:14:50:589 FINE Using optimized Accessor for public com.orientechnologies.orient.server.config.OServerCommandConfiguration[] com.orientechnologies.orient.server.config.OServerNetworkListenerConfiguration.commands [OptimizedAccessorFactory]
2015-01-02 21:14:50:590 FINE Using optimized Accessor for public com.orientechnologies.orient.server.config.OServerEntryConfiguration[] com.orientechnologies.orient.server.config.OServerCommandConfiguration.parameters [OptimizedAccessorFactory]
2015-01-02 21:14:50:591 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerEntryConfiguration.value [OptimizedAccessorFactory]
2015-01-02 21:14:50:591 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerEntryConfiguration.name [OptimizedAccessorFactory]
2015-01-02 21:14:50:592 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerCommandConfiguration.implementation [OptimizedAccessorFactory]
2015-01-02 21:14:50:592 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerCommandConfiguration.pattern [OptimizedAccessorFactory]
2015-01-02 21:14:50:593 FINE Using optimized TransducedAccessor for com.orientechnologies.orient.server.config.OServerCommandConfiguration#stateful [OptimizedTransducedAccessorFactory]
2015-01-02 21:14:50:594 FINE Using optimized Accessor for public com.orientechnologies.orient.server.config.OServerParameterConfiguration[] com.orientechnologies.orient.server.config.OServerNetworkListenerConfiguration.parameters [OptimizedAccessorFactory]
2015-01-02 21:14:50:595 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerNetworkListenerConfiguration.protocol [OptimizedAccessorFactory]
2015-01-02 21:14:50:595 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerNetworkListenerConfiguration.socket [OptimizedAccessorFactory]
2015-01-02 21:14:50:596 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerNetworkListenerConfiguration.portRange [OptimizedAccessorFactory]
2015-01-02 21:14:50:596 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerNetworkListenerConfiguration.ipAddress [OptimizedAccessorFactory]
2015-01-02 21:14:50:597 FINE Using optimized Accessor for public com.orientechnologies.orient.server.config.OServerStorageConfiguration[] com.orientechnologies.orient.server.config.OServerConfiguration.storages [OptimizedAccessorFactory]
2015-01-02 21:14:50:598 FINE Using optimized TransducedAccessor for com.orientechnologies.orient.server.config.OServerStorageConfiguration#loadOnStartup [OptimizedTransducedAccessorFactory]
2015-01-02 21:14:50:598 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerStorageConfiguration.userPassword [OptimizedAccessorFactory]
2015-01-02 21:14:50:599 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerStorageConfiguration.userName [OptimizedAccessorFactory]
2015-01-02 21:14:50:600 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerStorageConfiguration.path [OptimizedAccessorFactory]
2015-01-02 21:14:50:600 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerStorageConfiguration.name [OptimizedAccessorFactory]
2015-01-02 21:14:50:601 FINE Using optimized Accessor for public com.orientechnologies.orient.server.config.OServerUserConfiguration[] com.orientechnologies.orient.server.config.OServerConfiguration.users [OptimizedAccessorFactory]
2015-01-02 21:14:50:602 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerUserConfiguration.resources [OptimizedAccessorFactory]
2015-01-02 21:14:50:602 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerUserConfiguration.password [OptimizedAccessorFactory]
2015-01-02 21:14:50:603 FINE Using optimized Accessor for public java.lang.String com.orientechnologies.orient.server.config.OServerUserConfiguration.name [OptimizedAccessorFactory]
2015-01-02 21:14:50:603 FINE Using optimized Accessor for public com.orientechnologies.orient.server.config.OServerSecurityConfiguration com.orientechnologies.orient.server.config.OServerConfiguration.security [OptimizedAccessorFactory]
2015-01-02 21:14:50:604 FINE Using optimized Accessor for public java.util.List com.orientechnologies.orient.server.config.OServerSecurityConfiguration.users [OptimizedAccessorFactory]
2015-01-02 21:14:50:605 FINE Using optimized Accessor for public java.util.List com.orientechnologies.orient.server.config.OServerSecurityConfiguration.resources [OptimizedAccessorFactory]
2015-01-02 21:14:50:605 FINE Using optimized Accessor for public com.orientechnologies.orient.server.config.OServerEntryConfiguration[] com.orientechnologies.orient.server.config.OServerConfiguration.properties [OptimizedAccessorFactory]
2015-01-02 21:14:50:619 INFO Installing Script interpreter. WARN: authenticated clients can execute any kind of code into the server by using the following allowed languages: [sql] [OServerSideScriptInterpreter]
2015-01-02 21:14:50:619 INFO OrientDB Server v1.7.10 is active. [OServer]
2015-01-02 21:14:57:474 WARN Distributed server is not yet ONLINE (status=OFFLINE), reject incoming connection from /127.0.0.1:43482 [OServerNetworkListener]
```

**Comment by lvca** [comment]
Since it's fixed in 2.0, I consider it as closed

**Comment by lvca** [comment]
Since it's fixed in 2.0, I consider it as closed
